estate/models/estate_property.py

```diff
@@ -94,9 +104,10 @@ def _compute_best_offer(self):
 
     @api.onchange("garden")
     def _compute_garden(self):
-        if not property.garden:
-            property.garden_area = 0
-            property.garden_orientation = False
+        for property in self:
```

Comment:
Pas de loop dans un onchange

Reply:
Thanks, i checkout that out and confirm that : 
self in an onchange context will always contain only one record at a time (the one currently being edited)